### PR TITLE
add current profile annotations to CVO manifests

### PR DESCRIPTION
**Pull request by guillaumerose**
This is matches openshift/enhancements#414 and doesn't change existing behavior

### CODE REVIEW

File: manifests/02_operator_cr.yaml [hunk 1]
```diff
@@ -4,5 +4,6 @@ metadata:
   name: cluster
   annotations:
     release.openshift.io/create-only: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
```
Comment (deads2k) [inline manifests/02_operator_cr.yaml:7]: alphabetical please

Comment (guillaumerose) [reply manifests/02_operator_cr.yaml:6]: fixed, sorry.

**Comment by deads2k** [comment]
This is consistent with our default opt-in profile

/lgtm

/assign @jsafrane

for approval.

**Pull request merged**